host_vars/dev.gvl.org.au.yml

```diff
@@ -100,6 +100,8 @@ host_galaxy_config_files:
     dest: "{{ galaxy_config_dir }}/mail/activation-email.txt"
   - src: "{{ galaxy_config_file_src_dir }}/config/local_tool_data_table_conf.xml"
     dest: "{{ galaxy_config_dir }}/local_tool_data_table_conf.xml"
+  - src: "{{ galaxy_tool_box_filters_dir }}/proteomics.yml"
```

Comment:
this is not going to work. src should be w.r.t the infrastructure directory - galaxy_tool_box_filters_dir is on the destination VM